Acceleratio.XCellKit/ExcelExport/Chart/Chart.cs

```diff
@@ -0,0 +1,192 @@
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Drawing.Charts;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Spreadsheet;
+using System.IO;
+using System.Linq;
+using Acceleratio.XCellKit.ExcelExport.Helpers;
+using Chart = DocumentFormat.OpenXml.Drawing.Charts.Chart;
+
+namespace Acceleratio.XCellKit.ExcelExport
+{
+    public class Charts : ReportExport
+    {
+        public Charts(ReportExport export)
+            : base(export)
+        {
+
+        }
+        /// <summary>
+        /// Creates, designes and places chart into a sheet.
+        /// Calls overrideable methods for displaying the chart based on ChartType.
+        /// </summary>
+        public void InsertChartInSpreadsheet(Stream stream)
```

Comment:
can we make this a bit more generic? why not return a drawing part and let me do what i want with it. From the current usage i must create a excel file once, and then it is opened again here. If i were to have a drawing part as the output, i could append it anywhere in my excel document.